src/sentry/search/snuba/executors.py

```diff
@@ -491,24 +494,92 @@ def has_sort_strategy(self, sort_by: str) -> bool:
         return sort_by in self.sort_strategies.keys()
 
 
+@dataclasses.dataclass
+class BetterPriorityParams:
+    # (event or issue age_hours) / (event or issue halflife hours)
+    # any event or issue age that is greater than max_pow times the half-life hours will get clipped
+    max_pow: int
+    min_score: float  # apply a min on the individual scores to avoid multiplying by zeroes
+
+    # event-aggregate scoring
+    event_age_weight: int  # [1, 5]
+    log_level_weight: int  # [0, 10]
+    stacktrace_weight: int  # [0, 3]
+    event_halflife_hours: int  # halves score every x hours
+
+    # issue-aggregate scoring
+    issue_age_weight: int  # [1, 5]
+    issue_halflife_hours: int  # halves score every x hours
+    relative_volume_weight: int  # [0, 10]
+
+    #
+    v2: bool
+    normalize: bool
+
+
 def better_priority_aggregation(
     start: datetime,
     end: datetime,
     aggregate_kwargs: PrioritySortWeights,
 ) -> Sequence[str]:
-    issue_age_weight = 1  # [1, 5]
-    event_age_weight = 1  # [1, 5]
-    log_level_weight = aggregate_kwargs["log_level"]  # [0, 10]
-    stacktrace_weight = aggregate_kwargs["has_stacktrace"]  # [0, 3]
-    # (event or issue age_hours) / (event or issue halflife hours)
-    # any event or issue age that is greater than max_pow times the half-life hours will get clipped
-    max_pow = 16
-    min_score = 0.01
-    event_age_hours = "divide(now() - timestamp, 3600)"
-    event_halflife_hours = aggregate_kwargs["event_halflife_hours"]  # halves score every x hours
-    issue_halflife_hours = aggregate_kwargs["issue_halflife_hours"]  # halves score every x hours
-    issue_age_hours = "divide(now() - min(timestamp), 3600)"
+    return better_priority_aggregation_impl(
+        BetterPriorityParams(
+            max_pow=16,
+            min_score=0.01,
+            event_age_weight=1,
+            log_level_weight=aggregate_kwargs["log_level"],
+            stacktrace_weight=aggregate_kwargs["has_stacktrace"],
+            event_halflife_hours=aggregate_kwargs["event_halflife_hours"],
+            issue_age_weight=1,
+            issue_halflife_hours=aggregate_kwargs["issue_halflife_hours"],
+            relative_volume_weight=aggregate_kwargs["relative_volume"],
+            v2=aggregate_kwargs["v2"],
+            normalize=aggregate_kwargs["norm"],
+        ),
+        "timestamp",
+        True,
+    )
+
+
+def better_priority_platform_aggregation(
```

Comment:
Maybe `better_priority_issue_platform_aggregation` would be clearer, took me a second to realize what this was for.